tests/Server/TextDocument/CompletionTest.php

```diff
@@ -160,7 +160,12 @@ public function testNewInNamespace()
                 'TestClass',
                 CompletionItemKind::CLASS_,
                 null,
-                'Pariatur ut laborum tempor voluptate consequat ea deserunt.',
+                'Pariatur ut laborum tempor voluptate consequat ea deserunt.' . "\n\n" .
+            'Deserunt enim minim sunt sint ea nisi. Deserunt excepteur tempor id nostrud
+laboris commodo ad commodo velit mollit qui non officia id. Nulla duis veniam
+veniam officia deserunt et non dolore mollit ea quis eiusmod sit non. Occaecat
+consequat sunt culpa exercitation pariatur id reprehenderit nisi incididunt Lorem
+sint. Officia culpa pariatur laborum nostrud cupidatat consequat mollit.',
```

Comment:
please don't do un-indented multi-line strings. They break code folding in editors